content/desktop/faqs/linuxfaqs.md

```diff
@@ -6,7 +6,11 @@ aliases:
 - /desktop/linux/space/
 ---
 
-### What is the difference between Docker Desktop for Linux and Docker Engine
+### What is the difference between Docker Desktop for Linux and Docker Engine?
+
+Docker Desktop for Linux provides a user-friendly graphical interface that simplifies the management of containers and services. It includes Docker Engine as this is the core technology that powers Docker containers. Docker Desktop for Linux also comes with additional features like Docker Scout and Docker Extensions.
+
+## Can I have both Docker Desktop for Linux and Docker Engine installed on my machine?
```

Comment:
The level 2 title looks out of place. I'm not sure if this one should be level 3 or the previous one should also be level 2.